src/Modules/Isrs.asm

```diff
@@ -609,22 +609,8 @@ t2_int_exit:
 ;
 ; Timer3 interrupt routine
 ;
```

Comment:
The task of timer 3 is still commutation timing though - I think we can keep the "Tasks" documentation here, no?

Reply:
It is explained at Bluejay.asm line 43